.travis/run_python_tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+# Runs unit tests of exonum_launcher_java_plugins.
+
+# Fail immediately in case of errors and/or unset variables
+set -eu -o pipefail
+
+# Echo commands so that the progress can be seen in CI server logs.
+set -x
+
+cd "${TRAVIS_BUILD_DIR}/exonum-java-binding"
+
+# Generate protobuf files needed for plugins
+source ./tests_profile
+mvn install -DskipTests -DskipRustLibBuild -pl core -am
```

Comment:
Why install if you need to `generate-sources` (and no Java tests or Rust build will happen to skip)?

Reply:
I experienced issues with maven unable to resolve project dependencies when used `generate-sources` on Travis.

Reply:
That's unexpected — which exactly dependencies? 3rd party or a local subproject?

Reply:
Build log here: https://api.travis-ci.com/v3/job/287154757/log.txt